stl/inc/xutility

```diff
@@ -4667,6 +4667,63 @@ _NODISCARD bool lexicographical_compare(
 }
 #endif // _HAS_CXX17
 
+#if defined(__cpp_impl_three_way_comparison) && defined(__cpp_lib_concepts)
+// FUNCTION TEMPLATE lexicographical_compare_three_way
+template <class _InIt1, class _InIt2, class _Cmp>
+constexpr auto lexicographical_compare_three_way(_InIt1 _First1, _InIt1 _Last1, _InIt2 _First2, _InIt2 _Last2,

```

Comment:
This should be marked `_NODISCARD` because it's a pure observer. (That is, it wouldn't make sense to call this function and then discard the result, because the function has no other effects.)

Reply:
Done